reo/src/reopt.jl

```diff
@@ -757,6 +772,7 @@ function reopt_run(m, p::Parameter)
 	## Temporary workaround for outages TimeStepsWithoutGrid
 	if !isempty(p.TimeStepsWithoutGrid)
 		add_no_grid_constraints(m, p)
+		add_no_grid_export_constraint(m, p)
```

Comment:
would it make sense to combine add_no_grid_constraints and add_no_grid_export_constraints into one function if they're only ever together?

Reply:
I put them in separate functions because one of the speed tips from Oscar Dowson was to add model constraints in individual functions.